src/Operator/Constructors.jl

```diff
@@ -408,7 +408,7 @@ operator field:
 ```jldoctest
 # setup
 mesh = Mesh2D(1.0, 1.0);
-advection = GalleryVectorOperator("advection", 4, 4, 3, mesh);
+advection = GalleryVectorOperator("advection", 4, 4, 1, mesh);
```

Comment:
The `GalleryVectorOperator`, in general, should have multiple components.  I don't think we want 'vector' examples with only one component.

Reply:
That makes sense, but my tests would not pass when I revert back to ```GalleryVectorOperator("advection", 4, 4, 3, mesh)```